lib/simulation/openfisca/mapping/index.js

```diff
@@ -119,18 +119,23 @@ var applyRessources = function(mesAidesEntity, openFiscaEntity, mappingSchema, s
     });
 };
 
-var setNonInjectedPrestationsToZero = function(openFiscaEntity, dateDeValeur) {
+var setNonInjectedPrestationsToZero = function(famille, individus, dateDeValeur) {
     var periods = getPeriods(dateDeValeur);
     var targetPeriods = periods.last12Months.concat([periods.rollingYear]);
-
+    var prestations = _.pick(PRESTATIONS, function(definition) {
```

Comment:
I guess we need a domain term to depict this kind of prestations (where the benefit is an amount of money).
Prestation monétaire ? allocation ?